### ISSUE TITLE
Need moar microarray accessions!

### ISSUE THREAD

**Issue by kurtwheeler** [AlexsLemonade/refinebio#936]
### Context

We finished processing human, mouse, and rat microarray data and we're pretty blocked on RNA-Seq data. We want to keep things running so we need some more data to chew on.

### Problem or idea

In slack @cgreene mentioned doing more organisms. Do we want to choose certain additional organisms to target or just try to find a way to do all the remaining microarray data that was collected using chips made by manufacturers we are able to process?

### Solution or next step

What organisms should we go for? Do we have any scripts that could be re-used to help generate these lists programmatically?

### New Issue Checklist

- [x] The title is short and descriptive.
- [x] You have explained the context that led you to write this issue.
- [x] You have reported a problem or idea.
- [x] You have proposed a solution or next step.

**Comment by jaclyn-taroni** [comment]
According to #642, the next set of organisms was fly (`drosophila2`) and worm (`celegans`). Tagging @cansav09 for the second question.

**Comment by kurtwheeler** [comment]
We tore through human/mouse/rat microarray really fast. I think we should just knock out all the microarray data that we know we're gonna want eventually while we don't have much RNA-Seq data crunching.

**Comment by cansav09** [comment]
Yes, I do have a script that we can use to be generating these lists for us. I can put it somewhere and make it a little more automated.

**Comment by cansav09** [comment]
Just tell me where I should put the GEO accession list making scripts and how you would like them to work. The Array Express list download isn't super automated right now, but I might be able to make it that way.

**Comment by kurtwheeler** [comment]
Hmm, I don't have any strong preferences for where the scripts should go, but I guess we might as well make a repo for them so they can have source control. As far as how they should work goes, I guess we'd probably want to specify an organism name and then have it spit out all of the experiment accessions for that organism that use manufacturers we can process (affymetrix or illumina).

The only other way we might want it to work would be to just get ALL the remaining accessions for manufacturers we can process rather than just specific organisms. Idk if that would be too much data that isn't useful enough for it to be worth processing though. That'd be a call for @jaclyn-taroni and @cgreene

**Comment by jaclyn-taroni** [comment]
I believe the only thing we can't do in the absence of BrainArray is `NO_OP` (for the majority of non-Brainarray platforms, see https://github.com/AlexsLemonade/refinebio/issues/329) and it's handled by the SCAN processor (https://github.com/AlexsLemonade/refinebio/issues/64). Does that sound correct @kurtwheeler?

**Comment by kurtwheeler** [comment]
Wait that double negative is tripping me up. Are you basically saying that it doesn't matter if the microarrays are supported by BrainArray or not, except if they've already been processed because we can't NO_OP non-brainarray samples?

Wouldn't that mean that basically we just don't need to worry about whether or not a sample is supported by brainarray unless an experiment only has processed data? It can be hard to programmatically determine if an experiment only has processed data, doesn't have processed data, or a little of both, but our system knows how to do it. So if we just don't worry about brainarray our system can filter out the data that we aren't actually prepared to process.

**Comment by jaclyn-taroni** [comment]
Yes sorry I meant 1) we can process CEL files whether or not they are from a BrainArray supported platform. 2) we can’t do gene conversion for processed data for the majority non-BrainArray supported platforms.

I think we’re coming to the same conclusion which is we don’t need to filter the accessions to only those that are BrainArray supported. Do I understand correctly?

**Comment by kurtwheeler** [comment]
@cansav09 it looks like https://github.com/AlexsLemonade/accession_retrieval/pull/1 is still in progress, however it also looks like the discussion is primarily about how to make it more generally useful. It seems like this use case may already be supported as is. Have you used it to generate the list for this yet?

**Comment by cansav09** [comment]
The scripts aren't ready for use, but the list I produced: "arrays_not_done.csv" is ready for you to use. It should be the rest of the microarrays and not any arrays you've already processed.

**Comment by cansav09** [comment]
Also in regards to the other comments. I am working on making the scripts be more generalizable and useable. It will be sortable by keywords, which includes if you wanted to sort it by organism or etc. It also filters out non-supported platforms, but I've include an option so that you can also turn off that filter if you would like.

**Comment by kurtwheeler** [comment]
Perfect, thanks @cansav09 !

**Comment by kurtwheeler** [comment]
We ran all of these, this should have been closed a while ago.
